build.gradle

```diff
@@ -5,6 +5,7 @@ buildscript {
   repositories {
     google()
     jcenter()
+    gradlePluginPortal()
```

Comment:
What is this for?

Reply:
To disable animation in Espresso testing 
https://stackoverflow.com/questions/43751079/espresso-testing-disable-animation/43861179#43861179.

Reply:
Ah, can you add a brief comment in the Gradle file providing context & a link to that Stackoverflow answer? I think that would be helpful to other folks who run across this code and are unsure of what it's for.

Reply:
Yes sure I will add comment thanks.